plugins/restful/RestfulFilesUpload.php

```diff
@@ -26,6 +26,8 @@ public static function controllersInfo() {
    * - "validators": By default no validation is done on the file extensions or
    *   file size.
    * - "scheme": By default the default scheme (e.g. public, private) is used.
+   * - "make_permanent": When true the file's status will be set to permanent
```

Comment:
I like @amitaibu's suggestion. Of course, we will need to make `'set_usage'` to default to `'restful'`. That way we can omit it from the plugin definition.